openbb_terminal/core/session/session_model.py

```diff
@@ -88,7 +88,7 @@ def login(session: dict) -> LoginStatus:
         if response.status_code == 200:
             configs = json.loads(response.content)
             email = configs.get("email", "")
-            feature_settings = configs.get("features_settings", {})
+            feature_settings = configs.get("features_settings", {}) or {}
```

Comment:
?  is this just in case `features_settings` is empty and not an empty dict?

Reply:
Yes. I was getting an error because somehow the key "features_settings" is there but with the value None (maybe because I had values stored in the past but not anymore?) - causing the terminal to yield an error when later trying `feature_settings.get("FLAIR", None)`